### DISCUSSION TITLE
MacOS: tmux prefix 'option + a' have it instead printing 'å'

### DISCUSSION THREAD

**Discussion by geonnave** [ghostty-org/ghostty#3605] [Q&A] [answered]
When I press `option + a` (my custom tmux prefix), it prints the character `å` instead of activating the tmux prefix.

I am on MacOS Sonoma, my physical keyboard layout is US, and my keyboard language is set to ABC.

**Comment by rqel** [comment, marked as answer]
You can set `macos-option-as-alt` to `true` in your configuration, as explained in the documentation found [here](https://ghostty.org/docs/config/reference#macos-option-as-alt).

**Reply by geonnave** [reply]
Great it works, thanks!
